prow/config/config.go

```diff
@@ -913,27 +913,29 @@ func setPeriodicDecorationDefaults(c *Config, ps *Periodic) {
 // finalizeJobConfig mutates and fixes entries for jobspecs
 func (c *Config) finalizeJobConfig() error {
 	if c.decorationRequested() {
-		if c.Plank.DefaultDecorationConfig == nil {
-			return errors.New("no default decoration config provided for plank")
-		}
-		if c.Plank.DefaultDecorationConfig.UtilityImages == nil {
-			return errors.New("no default decoration image pull specs provided for plank")
-		}
-		if c.Plank.DefaultDecorationConfig.GCSConfiguration == nil {
-			return errors.New("no default GCS decoration config provided for plank")
+
+		if c.Plank.DefaultDecorationConfig != nil {
+			if len(c.Plank.DefaultDecorationConfigs) > 0 {
+				return errors.New("both default_decoration_config and default_decoration_configs are specified")
+			}
+
+			logrus.Warning("default_decoration_configs['*'] is missing, using the value from default_decoration_config instead.")
```

Comment:
This is where we should put the deprecation warning

Reply:
oh boy. wrong copy paste.

Reply:
fixed.